apps/nameguard.io/tsconfig.json

```diff
@@ -19,7 +19,9 @@
       }
     ],
     "paths": {
-      "@/*": ["./src/*"]
+      "@/*": ["./src/*"],
+      "@components": ["./src/components/"],
+      "@components/*": ["./src/components/*"]
```

Comment:
Can you help me understand why we need to include these additional paths? Thanks :smile:

Reply:
I deleted those 😄
I wanted to set up an alias for importing components using "@components/...". However, we already have a similar shortcut for the 'src' folder, so changing it from "@/components" to "@components" doesn't really help much.